packages/@adobe/spectrum-css-temp/components/menu/index.css

```diff
@@ -176,7 +176,8 @@ governing permissions and limitations under the License.
 }
 .spectrum-Menu-icon {
   grid-area: icon;
-  align-self: center;
+  vertical-align: top;
+  padding-block-start: calc(var(--spectrum-selectlist-option-text-size) / 7);
```

Comment:
Not very happy with this calculation, really the icon should have vertical padding matching the space above the span text (dependant on font size)

Reply:
Maybe something like ```padding-block-start: calc(calc(calc(var(--spectrum-body-4-text-line-height) - 1) * var(--spectrum-selectlist-option-text-size)) / 2);```? 

(Line height - 1) * font-size / 2? Assuming the space above and below the text is equal